Sources/Signal.swift

```diff
@@ -2360,69 +2338,63 @@ extension Signal {
 }
 
 extension Signal where Error == NoError {
-	/// Apply a failable `operation` to values from `self` with successful
-	/// results forwarded on the returned signal and thrown errors sent as
-	/// failed events.
+	/// Invoke a throwable action with every value from `self`, and forward the values
```

Comment:
I think _Apply a failable `operation`_ is clearer here.

_Invoke a throwable action_ sounds way too imperative.

c.f. the description of `map` from Swift:

> Returns an array containing the results of mapping the given closure over the sequence's elements.